kide:
  phidget_spatial_node:
    ros__parameters:
      publish_interval_ms: 1000
      motor_serials: [373316, 373225, 373306, 373257] #[373314]
